src/camera/arc_ball.rs

```diff
@@ -75,12 +75,12 @@ impl ArcBall {
             pitch: 0.0,
             dist: 0.0,
             min_dist: 0.00001,
-            max_dist: std::f32::MAX,
+            max_dist: 1.0e4,
             yaw_step: 0.005,
             pitch_step: 0.005,
             min_pitch: 0.01,
             max_pitch: std::f32::consts::PI - 0.01,
-            dist_step: 40.0,
+            dist_step: 1.02,
```

Comment:
I tried this and found a default `dist_step` of 1.01 to be smoother. The steps actually come in increments of 10 (because of [this line](https://github.com/sebcrozet/kiss3d/blob/7855eeedb8ba1af99caa15e7c140af6f67134476/src/window/gl_canvas.rs#L172) I believe), so 1.02 is really 1.02^10 ~= 1.22.